credshelper/index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html>
+<script>
+var config = {loginURL: "https://test.salesforce.com",
```

Comment:
Can we make this configurable or change it to `https://login.salesforce.com`? Our new org is on `GS0`, which is `Prod`.

Reply:
Good idea, I'll add some UI.